app/src/pages/inside/launchesPage/launchesPage.jsx

```diff
@@ -189,6 +174,14 @@ export class LaunchesPage extends Component {
       fetchFunc: this.props.fetchData,
     });
 
+  moveToDebugLaunches = (eventData) => {
```

Comment:
```
moveLaunchesToDebug = (eventData) => {
  const launches = eventData && eventData.id ? [eventData] : this.props.selectedLaunches;
  // ...
}
```

Reply:
Thank you for the suggestion. I will use a ternary operator more often
Fixed